modules/pipeline/spec/pipeline_task.go

```diff
@@ -278,6 +279,26 @@ func (pt *PipelineTask) GetMetadata() apistructs.Metadata {
 	return pt.Result.Metadata
 }
 
+type ExecutorChanData struct {
+	Data    interface{}
+	Version string
+}
+
+func (pt *PipelineTask) GenerateExecutorVersion() string {
```

Comment:
What is executor version? No, executor doesn't have version.
You should use `GenerateExecutorDoneChanDataVersion`.